pipeline/rpc/peer.go

```diff
@@ -10,7 +10,6 @@ type (
 	// Filter defines filters for fetching items from the queue.
 	Filter struct {
 		Labels map[string]string `json:"labels"`
-		Expr   string            `json:"expr"`
```

Comment:
you have to update the woodpecker.proto too

Reply:
Did that and updated the notes how to install use protoc as it was my first time using it and the old guide wasn't complete.

Reply:
well each time it gets better, before i touched it there was none ;) - thanks